### DISCUSSION TITLE
I'm getting error 404 after configuring the hostname, which could be the reason?

### DISCUSSION THREAD

**Discussion by qeepoo** [globaleaks/globaleaks-whistleblowing-software#4223] [Q&A] [answered]
### What version of GlobaLeaks are you using?

5.0.12

### What browser(s) are you seeing the problem on?

All

### What operating system(s) are you seeing the problem on?

Windows, macOS

### Describe the issue

Hello, I'm trying to create a docker container with globaleaks v5.0.12, however it gives my "Internal Server Error" after configuring the domain and certificate trough the web interface. Other times it gives me error 400.
I'm using Traefik as a proxy, which provides SSL-Offloading
Could somebody help?

**Comment by evilaliv3** [comment, marked as answer]
Hello @qeepoo 

Thank you for your question.

GlobaLeaks listens only on:
- via IP if no hostname is configured
- on the hostname configured in the root tenant or secondary tenants

If you are receiving error 400 it means that when the connection arrives to globaleaks, via traefik, it does not contains improper hostname.


Aside from this i've two questions:
- do you really need tls offloading in your setup? we advise to not use no proxy in relation to whistleblowing laws to try to keep the setup end2end like.
- are you aware we provide official docker images? https://hub.docker.com/repository/docker/globaleaks/globaleaks

**Comment by qeepoo** [comment]
Hi thanks for the quick response.
So after installing with the docker-compose and using the official image of globaleaks available on dockerhub, I access the instance trough an SSH tunnel, after which I complete the first install wizard.

After completing the first install wizard I access the web interface and configure the hostname of the default site, to match the one on my traefik labels.
 Note: after configuring the hostname, the instance becomes unavailable trough the SSH tunnel (IP+port).
 It doesn't matter if I configure the hostname or not, it gives me internal server error everytime.
 I also tried to generate the let's encrypt certificate however it gives me an "Unexpected Error".
 I also tried to upload the certificate used by traefik (let's encrypt), and it still doesn't work.
 
I have an instance running in dev which was one of my coworkers that deployed it about a year ago. I tried migrating that one to production, however it also gives me Internal Server Error, or error 400.

Responding to both your questions

do you really need tls offloading in your setup? we advise to not use no proxy in relation to whistleblowing laws to try to keep the setup end2end like. - We need to because our production server already runs a proxy and a bunch of other microservices. 
are you aware we provide official docker images? https://hub.docker.com/repository/docker/globaleaks/globaleaks - Yes, I initially tried building the image based on the Dockerfile but started using the official one.

I can provide some more details tomorrow, like the docker compose I'm using, print screens of the configuration I have aswell as the errors I mentioned above.

**Comment by qeepoo** [comment]
Ok managed to solve the issue. I added a few flags to the Traefik configuration to passthrough and terminate the TLS connection on the backend.
